tests/test_validation.py

```diff
@@ -0,0 +1,38 @@
+from django.core.exceptions import ValidationError
+
+import pytest
+
+from zgw_consumers.models.validators import NonUrlValidator, PrefixValidator
+
+
+def test_start_with_validator_starts_with_false():
+    validator = PrefixValidator(prefix="/", starts_with=False)
+
+    assert validator.__call__("no_leading_slash") is None
+
+    with pytest.raises(ValidationError) as exc_context:
+        validator.__call__("/with_leading_slash")
+
+    assert "The given value cannot start with '/'" in exc_context.value
```

Comment:
```suggestion
    with pytest.raises(
        ValidationError,
        match="The given value cannot start with '/'",
    ):
        validator("/with_leading_slash")
```
you can pass the (pattern) of the exception message directly :)